docs/API/chat.zh.md

```
@@ -293,12 +299,55 @@ Agent 需要用户做决策才能继续时下发。SSE 流随后暂停,直到通
 id: 5
 event: message
-data: {"type":"createMessage","payload":{"message_id":"act_0005","role":"assistant","content":[{"type":"markdown","payload":{"content":"销售额前 5 的客户:\n"}}]}}
+data: {"type":"createMessage","payload":{"message_id":"act_0005","role":"assistant","content":[{"type":"markdown","payload":{"content":"销售额前 5 的客户:\n"}}],"depth":0,"parent_action_id":null}}
+```
+
+Sub-agent 事件(`depth=1`):
+
+```
+id: 12
+event: message
+data: {"type":"createMessage","payload":{"message_id":"act_0012","role":"assistant","content":[{"type":"call-tool","payload":{"callToolId":"act_0012","toolName":"execute_sql","toolParams":{"sql":"SELECT 1"}}}],"depth":1,"parent_action_id":"act_0006"}}
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)